Configuration/StandardSequences/python/AlCaHarvesting_cff.py

```diff
@@ -248,15 +248,15 @@
 
 # --------------------------------------------------------------------------------------
 # PPS calibration
-ALCAHARVESTPPSTimingCalibration_metadata = cms.PSet(record = cms.untracked.string('PPSTimingCalibrationRcd'))
+ALCAHARVESTPPSTimingCalibration_metadata = cms.PSet(record = cms.untracked.string('PPSTimingCalibrationRcd_HPTDC'))
 ALCAHARVESTPPSTimingCalibration_dbOutput = cms.PSet(record = cms.string('PPSTimingCalibrationRcd'),
```

Comment:
wouldn't you have to change the record to `PPSTimingCalibrationRcd_HPTDC` here as well?